libs/blocks/share/share.js

```diff
@@ -20,6 +20,7 @@ export async function getSVGsfromFile(path, selectors) {
   return selectors.map((selector) => {
     const symbol = doc.querySelector(`#${selector}`);
     const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+    if (!symbol) return null;
```

Comment:
nit: move up a line

Reply:
@So1ene can you address so we can merge this?